source/fe/fe_dgq.cc

```diff
@@ -694,6 +696,16 @@ FE_DGQ<dim, spacedim>::compare_for_domination(
       else
         return FiniteElementDomination::other_element_dominates;
     }
+  else if (const Simplex::FE_DGP<dim, spacedim> *fe_dgp_other =
+             dynamic_cast<const Simplex::FE_DGP<dim, spacedim> *>(&fe_other))
```

Comment:
Why is here only `Simplex::FE_DGP` and not the version for pyramids and wedges?

Reply:
May I ask you to add a test.

Reply:
You've checked only for certain elements in assertions in `compare_for_domination` inside `fe_lib.cc`.

I've only added corresponding conditionals to the counterparts marked in these assertions.

I'm not happy that we need to declare the domination relations in both elements. There should be one large construct that knows about all FE relations. But for now let's stick to this.

Reply:
Oh, there are currently no functions `FE_WedgeDGP::compare_for_domination()` and `FE_PyramidDGP::compare_for_domination()`. I will add them.

Reply:
Do you still need to update this one place now with the other kinds of elements?